lib/temporal/metadata/workflow_task.rb

```diff
@@ -3,16 +3,18 @@
 module Temporal
   module Metadata
     class WorkflowTask < Base
-      attr_reader :namespace, :id, :task_token, :attempt, :workflow_run_id, :workflow_id, :workflow_name
+      attr_reader :namespace, :id, :task_token, :attempt, :workflow_run_id, :workflow_id, :workflow_name, :query_type, :query_args
 
-      def initialize(namespace:, id:, task_token:, attempt:, workflow_run_id:, workflow_id:, workflow_name:)
+      def initialize(namespace:, id:, task_token:, attempt:, workflow_run_id:, workflow_id:, workflow_name:, query_type:, query_args:)
         @namespace = namespace
         @id = id
         @task_token = task_token
         @attempt = attempt
         @workflow_run_id = workflow_run_id
         @workflow_id = workflow_id
         @workflow_name = workflow_name
+        @query_type = query_type
+        @query_args = query_args
```

Comment:
This smells like a "data clamp" and might benefit from our own `Query` struct combining the two together